src/gradescopeapi/_config/config.py

```diff
@@ -2,63 +2,54 @@
 Configuration file for FastAPI. Specifies the specific objects and data models used in our api
 """
 
-import requests
-from enum import Enum
+from datetime import datetime
+from typing import Optional
 from pydantic import BaseModel
 
 
-class SemesterModel(str, Enum):
-    spring = "spring"
-    summer = "summer"
-    fall = "fall"
-    winter = "winter"
+class LoginRequestModel(BaseModel):
+    email: str
+    password: str
 
 
-class SessionModel(BaseModel):
-    session: requests.Session
+class CourseID(BaseModel):
+    course_id: str
 
 
-class ConnectionModel(BaseModel):
-    session: SessionModel
-    logged_in: bool = False
+class AssignmentID(BaseModel):
+    course_id: str
+    assignment_id: str
 
 
-class TokenModel(BaseModel):
-    access_token: str
+class AssignmentDates(BaseModel):
+    course_id: str
+    assignment_id: str
+    release_date: Optional[datetime] = None
+    due_date: Optional[datetime] = None
+    late_due_date: Optional[datetime] = None
```

Comment:
I'm not that familiar with fastapi but how does a user send an HTTP request with a python object like a datetime?